dart/pom.xml

```diff
@@ -35,6 +35,18 @@
       <artifactId>compile-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.parceler</groupId>
+      <artifactId>parceler-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+
+    <dependency>
+        <groupId>com.squareup</groupId>
+        <artifactId>javapoet</artifactId>
```

Comment:
Is this absolutely required? Since Dart doesn't have seperate artifacts for the annotation processing and runtime API, we don't want to add these dependencies to the the user's app.